jax/experimental/jax2tf/README.md

```diff
@@ -78,9 +78,52 @@ tf.saved_model.save(my_model, '/some/directory')
 restored_model = tf.saved_model.load('/some/directory')
 ```
 
+To explain (TODO): 
+
+  * saved model and custom gradients
+  * saved model and XLA
+
 ### Running on GPU
 
 To run jax2tf on GPU, both jaxlib and TensorFlow must be installed with support
 for CUDA. One must be mindful to install a version of CUDA that is compatible
 with both [jaxlib](../../../../../#pip-installation) and
 [TensorFlow](https://www.tensorflow.org/install/source#tested_build_configurations).
+
+## Known limitations
+
+#### Errors due to nested invocations of XLA
```

Comment:
Have you filed an issue against TF for this (or is there an existing issue?)

Reply:
I have filed a bug with TF. There are already other bugs in the area of nested compilations.